securedrop_client/db.py

```diff
@@ -215,6 +215,18 @@ def seen_by(self, journalist_id: int) -> bool:
 
         return False
 
+    @property
+    def seen_by_list(self) -> Dict[str, "User"]:
+        """
+        For each message retrieve a dictionary of users who have seen it.
+        Each dictionary item consists of the user's username as its key and the user
+        object as its value.
+        """
+        usernames = {}  # type: Dict[str, 'User']
```

Comment:
Note (not a blocker at all): the type hint in comments don't need to be string literals.